src/azul/csp.py

```diff
@@ -37,7 +37,7 @@ def for_azul(cls, nonce: str | None = None) -> Self:
             'default-src': [self],
             'img-src': [self, data],
             'script-src': [self, *nonce],
-            'style-src': [self, *nonce],
+            'style-src': [self],
```

Comment:
I think this should be left unchanged. If there were inline styles, they would need a nonce.